### ISSUE TITLE
SNAC API "search" with entity_type = corporateBody yields in no results

### ISSUE THREAD

**Issue by DiegoPino** [snac-cooperative/snac#298]
**Describe the bug**
When calling the API using this 

`body:`

```JSON
{"command":"search","term":"IBM","entity_type":"corporateBody","start":0,"count":10}
```
No results come back. `personal` and `family` work well

**To Reproduce**
Use curl and call
```Shell
curl https://api.snaccooperative.org -XPUT -H "Content-Type: application/json"  --data '{"command":"search","term":"IBM","entity_type":"corporateBody","start":0,"count":10,"search_type":"autocomplete"}' 
```

You will get no results.

**Expected behavior**
Same as with `person` or `family` we should see results

**Screenshots**
Does not apply

**Browser**
Does not apply

**Comment by glassjoseph** [comment]
Thanks for the bug report. Looks like it was a few lines of code that were incompatible with a recent package upgrade. Should be fixed now by #299!

**Comment by DiegoPino** [comment]
@glassjoseph thanks so much!
